packages/library-base/src/commonMain/kotlin/io/realm/dynamic/DynamicMutableRealm.kt

```diff
@@ -56,6 +56,8 @@ public interface DynamicMutableRealm : DynamicRealm {
      * if the object graph contains an object from a previous version or if a property does not
      * match the underlying schema.
      */
+    // NOTE Doesn't prevent copying embedded objects, but theres not really a good way to
+    //      differentiate the dynamic objects without bloating the type space
```

Comment:
It would require that we could differentiate normal and embedded objects in the dynamic type system. But we deemed that it wouldn't be worth it, because we are already differentiating mutable and immutable ones, so the type system would be bloated.